tests_integration/cli/config/editorconfig/file.js

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable */
```

Comment:
eslint-disable comments shouldn't be needed because we ignore test_integration/cli/ in .eslintignore